app/src/main/java/openfoodfacts/github/scrachx/openfood/jobs/FileDownloader.java

```diff
@@ -13,23 +13,39 @@
 
 import java.io.*;
 
+/**
+ * File Downloader class which is used to download a file and
+ * write response to the disk.
+ */
 public class FileDownloader {
     private Context context;
 
     public interface FileReceiver {
         void fileDownloaded(File file);
     }
 
+    /**
+     * Constructor of the class used to initialize the objects.
+     * @param context : context to be set
+     */
```

Comment:
please add info about context: could it be null ?
If not add @NonNull annotation.